diff --git a/node_modules/@types/rebass/LICENSE b/node_modules/@types/rebass/LICENSE
old mode 100644
new mode 100755
diff --git a/node_modules/@types/rebass/README.md b/node_modules/@types/rebass/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/@types/rebass/index.d.ts b/node_modules/@types/rebass/index.d.ts
old mode 100644
new mode 100755
index 26ece82..d7c6ad3
--- a/node_modules/@types/rebass/index.d.ts
+++ b/node_modules/@types/rebass/index.d.ts
@@ -32,6 +32,12 @@ export interface BaseProps extends React.RefAttributes<any> {
  */
 export type SxStyleProp =
     | SystemStyleObject
+    | StyledSystem.SpaceProps<StyledComponents.DefaultTheme>
+    | StyledSystem.TypographyProps<StyledComponents.DefaultTheme>
+    | StyledSystem.FlexboxProps<StyledComponents.DefaultTheme>
+    | StyledSystem.GridProps<StyledComponents.DefaultTheme>
+    | StyledSystem.LayoutProps<StyledComponents.DefaultTheme>
+    | StyledSystem.ColorProps<StyledComponents.DefaultTheme>
     | Record<
           string,
           | SystemStyleObject
@@ -48,10 +54,10 @@ export interface SxProps {
 
 interface BoxKnownProps
     extends BaseProps,
-        StyledSystem.SpaceProps,
+        StyledSystem.SpaceProps<StyledComponents.DefaultTheme>,
         StyledSystem.LayoutProps,
-        StyledSystem.FontSizeProps,
-        StyledSystem.ColorProps,
+        StyledSystem.FontSizeProps<StyledComponents.DefaultTheme>,
+        StyledSystem.ColorProps<StyledComponents.DefaultTheme>,
         StyledSystem.FlexProps,
         StyledSystem.OrderProps,
         StyledSystem.AlignSelfProps,
diff --git a/node_modules/@types/rebass/styled-components.d.ts b/node_modules/@types/rebass/styled-components.d.ts
old mode 100644
new mode 100755
